### DISCUSSION TITLE
Weird theme issue with i3 x Kvantum x Orchis Dark theme

### DISCUSSION THREAD

**Discussion by SuperSecureHuman** [i3/i3#4702] [Q&A]
First of all, I am kinda not sure what to blame here - i3/Kvantum/Orchis/Picom

I am using kvantum with orchis theme. I am getting a weird border issue with this

![image](https://user-images.githubusercontent.com/88489071/143856545-e09fb5d8-3e35-4754-9588-ae538e799da0.png)

VLC uses kvantum theme

For some reason, this issue is even there in rofi

![image](https://user-images.githubusercontent.com/88489071/143857016-6f58cd85-73cb-4559-9175-05869d0ff16c.png)

I have used this theme back when I used KDE, and did not have this issue

**Comment by Airblader** [comment]
i3 does not support rounded corners. See also https://github.com/Airblader/i3/pull/294 for forks.
